tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,27 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_data = {
+            'name': 'Bugzilla Lite',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_data)
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % self.test_data['url'])
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
+        self.apps.uninstall('Bugzilla Lite')
+        self.push_resource('IMG_0001.jpg')
+
+    def test_fill_new_bug(self):
+        bugzilla_lite = BugzillaLite(self.marionette)
+        bugzilla_lite.launch()
+        bugzilla_lite.dismiss_tooltip()
+        bugzilla_lite.create_new_bug(self.testvars['fillbug']['title'], self.testvars['fillbug']['description'])
```

Comment:
Parameterizing the title and the description is not a bad idea, at first. It's a bit overkill though. I don't think anybody will want to change these parameters (unlike the login).
Hence I propose to define some variable instead:
``` python
time_of_test_run = time.time()
bug_title = 'GAIATEST_Title {}'.format(time_of_test_run)
bug_description = 'GAIATEST_Description {}'.format(time_of_test_run)

bugzilla_lite.create_new_bug(bug_title, bug_description) 
```

Using the epoch timestamp here, allows us to have a (nearly) unique identifier, so we can track which test created which bug. We use the [same trick with the SMS app](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms.py#L18), for instance.